api/models/constants.py

```diff
@@ -90,3 +90,16 @@ class CoordinatorGroupLimit(IntEnum):
     40: "B1",
     60: "B2",
 }
+
+
+TEACHER_PEER_SUPPORT_OPTIONS = [
+    "can_check_syllabus",
+    "can_host_mentoring_sessions",
+    "can_give_feedback",
+    "can_help_with_childrens_groups",
+    "can_provide_materials",
+    "can_invite_to_class",
+    "can_work_in_tandem",
+]
+
+TEACHER_PEER_SUPPORT_FIELD_NAME_PREFIX = "peer_support_"
```

Comment:
It makes sense to switch these constants in code: both alphabetically and logically

Reply:
Done